gunicorn/util.py

```diff
@@ -232,33 +232,30 @@ def is_ipv6(addr):
     return True
 
 
-def parse_address(netloc, default_port=8000):
+def parse_address(netloc, default_port='8000'):
+
     if re.match(r'unix:(//)?', netloc):
         return re.split(r'unix:(//)?', netloc)[-1]
 
     if netloc.startswith("tcp://"):
         netloc = netloc.split("tcp://")[1]
 
-    # get host
+    host, port = [netloc, default_port]
```

Comment:
No need to create a new list here. `host, port = netloc, default_port` should work, right?